packages/legacy-workbench/src/app.js

```diff
@@ -218,6 +220,21 @@ const moduleDefinition = function (productInfo) {
         ThemeService.applyDarkThemeMode();
 
         GuidesService.init();
+
+        // =========================
+        // Functions and configurations for integration with the shared-components module.
+        // =========================
+        let globalContextService = WorkbenchServiceProvider.get(WorkbenchGlobalContextService);
+
+        let languageChangeSubscriptions = globalContextService.onLanguageChanged()
```

Comment:
Again. Why we have a language service if we don't even use it?

Reply:
Refactored